docs/examples/clojure-example.md

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/codemirror.css" />
+
+# sample Clojure code
+
+``` clj
+(map inc '(1 2 3 4 5 6 7 8 9 10))
+```
+
+<script type="text/javascript">window.klipse_settings = {selector: '.lang-clj'};</script>
```

Comment:
`'.lang-clj'` => `'.language-clj'`, otherwise it worked great!

Reply:
I have modified as follows:
```suggestion
<script type="text/javascript">window.klipse_settings = {selector: '.lang-clj'};</script>
```

Reply:
I've modified as follows:
```suggestion
<script type="text/javascript">window.klipse_settings = {selector: '.lang-clj'};</script>
```